src/cellularnoise.rs

```diff
@@ -0,0 +1,100 @@
+use rand::*;
+use std::io::*;
+
+byond_fn! { cnoise_generate(precentage,smoothing_iterations, birth_limit, death_limit) {
+    noise_gen(precentage, smoothing_iterations, birth_limit, death_limit).ok()
+} }
+
+fn noise_gen(prec_as_str : &str, smoothing_level_as_str : &str, birth_limit_as_str : &str, death_limit_as_str : &str)-> Result<String> {
+    let prec = prec_as_str.parse::<i32>().expect("parse failed");
+    let smoothing_level = smoothing_level_as_str.parse::<i32>().expect("parse failed");
+    let birth_limit = birth_limit_as_str.parse::<i32>().expect("parse failed");
+    let death_limit = death_limit_as_str.parse::<i32>().expect("parse failed");
+    //Noise generation
+
+    let mut zplane = vec![vec![0; 254]; 254]; // 254 but we start at 0, and since byond starts at one it is 255 byond wise.
```

Comment:
- Should be `255`, not `254`
- Shouldn't be hardcoded to 255
- Can use `vec![0u8; ...]` to reduce memory footprint from default of `i32`, or even `vec![false; ...]` for additional clarity

Reply:
it should be 254 because our maps are 255x255

Reply:
agree with the rest though